main.tf

```diff
@@ -122,6 +122,17 @@ resource "aws_cloudwatch_event_target" "this" {
     }
   }
 
+  dynamic "http_target" {
+    for_each = lookup(each.value, "http_target", null) != null ? [
+      each.value.http_target
+    ] : []
+
+    content {
+      query_string_parameters = http_target.value.query_string_parameters
```

Comment:
There are 3 arguments supported according to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_event_target

Also, all of them should be optional.

Reply:
Clearly didn't pay attention there...

Reply:
No problems! Looks good to me now :)